data/ecosystems/c/celo.toml

```diff
@@ -38,6 +40,7 @@ sub_ecosystems = [
   "Curve",
   "DAO Maker",
   "Decentology",
+  "DefiLlama",
```

Comment:
we keep DefiLlama as general with no sub-ecos since it covers all crypto

Reply:
Got it. I put it in because Llamaswap supports Celo. Perhaps we can just include that somehow?

Reply:
OK, found a repo for llamaswap:

https://github.com/LlamaSwap

Strange that it wasn't part of the DefiLlama organization.